### PR TITLE
[buildbot] Fixes for embedded

### PR DESCRIPTION
**Pull request by adydychk**
- Changed name of Buildbot page (for private mode) `MediaSDK Private`
- Add variable `MASTER_PRODUCT_TYPE`
- Disabled `waterfall_view` and `grid_view`
- Add worker names for private buildbot

### CODE REVIEW

File: bb/master/config.py [hunk 1]
```diff
@@ -68,9 +69,11 @@ class Mode(Enum):
     BUILDBOT_URL = "http://mediasdk.intel.com/buildbot/"
 
 elif CURRENT_MODE == Mode.PRODUCTION_MODE_EMBEDDED:
```
Comment (adydychk) [inline bb/master/config.py]: Rename `Mode.PRODUCTION_MODE_EMBEDDED` to `Mode.PRODUCTION_MODE_PRIVATE`

File: bb/master/config.py [hunk 2]
```diff
@@ -60,17 +61,19 @@ class Mode(Enum):
...
+#CURRENT_MODE = Mode.PRODUCTION_MODE_PRIVATE
 #CURRENT_MODE = Mode.TEST_MODE
 
 if CURRENT_MODE == Mode.PRODUCTION_MODE:
     GITHUB_OWNERS_REPO = "MediaSDK"
     BUILDBOT_URL = "http://mediasdk.intel.com/buildbot/"
 
-elif CURRENT_MODE == Mode.PRODUCTION_MODE_EMBEDDED:
+elif CURRENT_MODE == Mode.PRODUCTION_MODE_PRIVATE:
+    BUILDBOT_TITLE = "Intel® MSDK-private"
```
Comment (azhogov) [inline bb/master/config.py]: MediaSDK Private

Comment (adydychk) [inline bb/master/config.py]: Change name to `MediaSDK private`

File: bb/master/msdk_secrets.py.example [hunk 3]
```diff
@@ -24,7 +24,7 @@ DATABASE_PASSWORD = "my_password"
 GITHUB_TOKEN = "my_token"
 
 # For embedded
-WORKERS = {BUILD: {"my_buildbox_name": {}},
-           TEST: {"my_testbox_name": {}}}
+WORKERS = {"build": {"my_buildbox_name": {}},
```
Comment (adydychk) [inline bb/master/msdk_secrets.py.example]: Can be in `config.py`

**Pull request merged**